README.rst

```diff
@@ -29,12 +29,12 @@ Supported versions
 ******************
 
 * Python 3.4, 3.5, 3.6, 3.7
-* Django 1.11, 2.0, 2.1
-* Django REST Framework 3.8
+* Django 1.11, 2.0, 2.1, 2.2
+* Django REST Framework 3.8, 3.9
```

Comment:
There's also DRF 3.7 in the test matrix, perhaps add in here? Alternatively remove it from the test matrix.